src/h5web/vis-packs/core/heatmap/config.tsx

```diff
@@ -84,18 +79,8 @@ function initialiseStore(onRehydrated: () => void) {
   );
 }
 
-const context = createContext<UseStore<HeatmapConfig> | undefined>(undefined);
-const { Provider } = context;
-
-export function useHeatmapConfig<U>(
-  selector: StateSelector<HeatmapConfig, U>,
-  equalityFn?: EqualityChecker<U>
-): U {
-  const useStore = useContext(context);
-  assertDefined(useStore);
-
-  return useStore(selector, equalityFn);
-}
+const { Provider, useStore } = createContext<HeatmapConfig>();
+export const useHeatmapConfig = useStore;
```

Comment:
The new `createContext` function allows us to remove our own config hooks and improves the API of the `Provider` component with an `initialStore` prop.